### PR TITLE
QOLOE-109 Print

### PR DESCRIPTION
**Pull request by tedqgov**
SWE Github print styles code: https://github.com/qld-gov-au/web-template-release

### CODE REVIEW

**Comment by ghazal-salehi** [comment]
Just needs to resolve conflicts in above files.

**Comment by tedqgov** [comment]
Okay, merge conflicts are resolved and pushed. Thanks @ghazal-salehi :)

File: .github/workflows/update.yml
Comment (stvp-qld) [inline .github/workflows/update.yml:1]: Hi @tedqgov, just checking re this file. Did you make changes to this as part of your commit?

**Review by stvp-qld — CHANGES_REQUESTED** [review]
Hi @tedqgov, just checking re this file. Did you make changes to this as part of your commit?

**Comment by duttonw** [comment]
please rebase this branch instead of merge develop in
git pull origin develop --rebase
at each conflict, review and fix then
git rebase --continue
after all is done, do a force push on this branch, it should only have your own commits all together.

**Review by stvp-qld — APPROVED** [review]

**Comment by tedqgov** [comment]
> please rebase this branch instead of merge develop in git pull origin develop --rebase at each conflict, review and fix then git rebase --continue after all is done, do a force push on this branch, it should only have your own commits all together.

Yup, I've only rebased so far instead of merge develop.

**Pull request merged**